sdk/aqueduct/aqueduct_client.py

```diff
@@ -417,12 +418,21 @@ def trigger(
         flow_id = parse_user_supplied_id(flow_id)
         self._api_client.refresh_workflow(flow_id, serialized_params)
 
-    def delete_flow(self, flow_id: Union[str, uuid.UUID]) -> None:
+    def delete_flow(
+        self,
+        flow_id: Union[str, uuid.UUID],
+        writes_to_delete: Optional[DefaultDict[uuid.UUID, List[WrittenObject]]] = None,
```

Comment:
I feel we could relate to 'save' which is a concept available for the rest of sdk. For example, `saved_objects_to_delete`. cc @kenxu95 @vsreekanti

Reply:
yeah thats fine with me. I agree, 'save' exposes the user to the same terminology as our API.